test/smoke/docker_smoke.sh

```diff
@@ -0,0 +1,6 @@
+#!/usr/bin/env bash
```

Comment:
I would love to know what each of these lines did.  I find it hard to parse docker configs.   Particularly lines 5 & 6.   I guess I expect we would cd to a workspace, install backstop locally to that space and use the feature tests runner.   I don't understand why `/test/configs/linux/node_modules` is called out in there.